modules/smart-agent_health-checker/conf/00-health-checker-value.yaml

```diff
@@ -0,0 +1,14 @@
+module: health-checker
```

Comment:
In general, the `00-` file is for heartbeat, if you can increment this please

Reply:
done